### ISSUE TITLE
$li not picking up insights spent at vendor

### ISSUE THREAD

**Issue by KyrathH** [Maselkov/GW2Bot-Red#9]
great job btw! not sure if I'm missing something but the $li command doesn't appear to pick up on the 150 insights I spent buying envoy insigs at Glenna, and subsequently crafting into a set of light precursor armour. I guess it would pick them up again if that set is upgraded to legendary (which I plan to do), but some people may choose not to do that.

**Comment by Maselkov** [comment]
I'm almost certain $li counts Envoy Insignia and precursor armors. Can you check again?

**Comment by KyrathH** [comment]
I have done, several times without any joy :( 
I have currently earned 394 insights, I used 150 on Envoy Insignia, which I then crafted into a set of light armour. I have completed the t2 collection and earned my chest of precursor armour but haven't picked a set yet (planning to use it for medium). I have crafted 6 Gifts of Prowess, using up another 150. And I have 94 li's left over in my inventory. Total 394. And yet the readout is as per the attached screencap :(
![gw2bot insights 14-6-17](https://user-images.githubusercontent.com/29379150/27146258-9dfeddb6-5130-11e7-8aab-3ff15a384869.jpg)
